test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_mnist.py

```diff
@@ -76,30 +86,52 @@ def test_distributed_mnist_no_ps(sagemaker_session, ecr_image, instance_type, fr
 @pytest.mark.multinode(2)
 @pytest.mark.integration("parameter server")
 def test_distributed_mnist_ps(sagemaker_session, ecr_image, instance_type, framework_version):
+    validate_or_skip_test(ecr_image=ecr_image)
     print('ecr image used for training', ecr_image)
     resource_path = os.path.join(os.path.dirname(__file__), '..', '..', 'resources')
-    script = os.path.join(resource_path, 'mnist', 'mnist_estimator2.py')
+    script = os.path.join(resource_path, 'mnist', 'mnist_estimator.py')
     estimator = TensorFlow(entry_point=script,
                            role='SageMakerRole',
-                           #hyperparameters={'sagemaker_parameter_server_enabled': True},
-                           instance_count=1,
+                           hyperparameters={'sagemaker_parameter_server_enabled': True},
+                           instance_count=2,
+                           instance_type=instance_type,
+                           sagemaker_session=sagemaker_session,
+                           image_uri=ecr_image,
+                           framework_version=framework_version)
+    inputs = estimator.sagemaker_session.upload_data(
+        path=os.path.join(resource_path, 'mnist', 'data-distributed'),
+        key_prefix='scriptmode/mnist-distributed')
+    estimator.fit(inputs, job_name=unique_name_from_base('test-tf-sm-distributed-mnist'))
+    _assert_checkpoint_exists(sagemaker_session.boto_region_name, estimator.model_dir, 200)
+
```

Comment:
nit: we use black code formatter (https://pypi.org/project/black/) - generally there should be two new lines between functions. You can execute it with

```
pip install black
black -l 120 /path/to/file.py
```